datafusion/sqllogictest/src/engines/datafusion_engine/normalize.rs

```diff
@@ -240,7 +240,8 @@ pub fn cell_to_string(col: &ArrayRef, row: usize) -> Result<String> {
                 Ok(cell_to_string(dict.values(), key)?)
             }
             _ => {
-                let f = ArrayFormatter::try_new(col.as_ref(), &DEFAULT_FORMAT_OPTIONS);
+                let f =
+                    ArrayFormatter::try_new(col.as_ref(), &DEFAULT_CLI_FORMAT_OPTIONS);
```

Comment:
This change makes it consistent with
https://github.com/apache/datafusion/blob/61afb0d486b4617ddc79cb01e8765fedd3ae247b/datafusion/sqllogictest/src/engines/datafusion_engine/normalize.rs#L198-L201